examples/cinder-csi-plugin/snapshot/example.yaml

```diff
@@ -11,6 +11,8 @@ kind: VolumeSnapshotClass
 metadata:
   name: csi-cinder-snapclass
 snapshotter: cinder.csi.openstack.org
+parameters:
+  force-create: "false"
```

Comment:
going the k8s way, `forceCreate` would be better.

Reply:
this has been discussed above...

Reply:
I mean prefer camel case in yaml, rather than kebab case.

Reply:
@jichenjc discussion is lost because it is seperate PR now . @adisky pls check here , https://github.com/kubernetes/cloud-provider-openstack/pull/1002 , camel case is for GO lang not for yamls

Reply:
my bad... I forgot this totally, @ramineni thanks for the reminder

Reply:
got it, thanks.